Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.9.1-stretch
```

Comment:
Suggestion: you could use multistage builds (i.e. build the app first, then use the output from the build (the binary) and put only that binary in an image. It makes for way smaller/faster images. Here are some[ references with examples](https://docs.docker.com/engine/userguide/eng-image/multistage-build/#before-multi-stage-builds).

Reply:
@aranajhonny Check the suggestion